docs/tutorials/five-minutes-to-feature-flags.md

```diff
@@ -0,0 +1,188 @@
+https://github.com/open-feature/five-minutes-to-feature-flags/blob/main/03_openfeature.js
+```
+
+We’ve imported the `@openfeature/js-sdk` NPM module, and used it to create an OpenFeature client called `featureFlags`. We then call `getBooleanValue` on that client to find out if the `with-cows` feature flag is true or false. Depending on what we get back we either show the new cow-based output, or the traditional plaintext format.
+
+Start the server with `node 03_openfeature.js`.
+
+```bash
+$> curl http://localhost:3333
+Hello, world!
+```
+
+:::note
+
+When we call `getBooleanValue` we also provide a default value of false. Since we haven’t configured the OpenFeature SDK with a _feature flag provider_ yet, it will always return that default value.
+
+:::
+
+## Configuring an OpenFeature Provider
+
+Without a feature flagging provider OpenFeature is pretty pointless - it’ll just return default values. Instead we want to connect our OpenFeature SDK to a full-fledged feature flagging system - a commercial product, an open-source system, or perhaps a custom internal system - so that it can provide flagging decisions from that system.
+
+Connecting OpenFeature to one of these backends is very straightforward, but it does require that we have an actual flagging framework set up. For now, let’s just configure a really, really simple provider that doesn’t need a backend:
+
+```js reference title="04_openfeature_with_provider.js"
+https://github.com/open-feature/five-minutes-to-feature-flags/blob/main/04_openfeature_with_provider.js
+```
+
+This minimalist provider is exactly that - you give it a hard-coded set of feature flag values, and it provides those values via the OpenFeature SDK.
+
+In our `FLAG_CONFIGURATION` above we’ve hard-coded that `with-cows` feature flag to true, which means that conditional predicate in our express app will now evaluate to true, which means that our service will now start providing bovine output. Start the server with `node 04_openfeature_with_provider.js` and test it out.
+
+```bash
+$> curl http://localhost:3333
+ _______________
+< Hello, world! >
+ ---------------
+        \   ^__^
+         \  (oo)\_______
+            (__)\       )\/\
+                ||----w |
+                ||     ||
+```
+
+If we changed that `with-cows` value to false, we’d see the more boring response:
+
+```bash
+$> curl http://localhost:3333
+Hello, world!
+```
+
+## Moving to a full feature-flagging system
+
+We’ve gotten started with OpenFeature using a very simple but extremely limited provider. The beauty of OpenFeature is that we can transition to a real feature-flagging system when we’re ready, without any change to how we evaluate flags. It’s as simple as configuring a different provider.
+
+**For example:**
+
+### CloudBees
+
+```js
+import { CloudbeesProvider } from 'cloudbees-openfeature-provider-node';
+
+const appKey = '[YOUR_APP_KEY]';
+OpenFeature.setProvider(await CloudbeesProvider.build(appKey));
+```
```

Comment:
Collapsable sections probably would be nice here

````suggestion
**For example:**

<details>
<summary>CloudBees</summary>

```js
import { CloudbeesProvider } from 'cloudbees-openfeature-provider-node';

const appKey = '[YOUR_APP_KEY]';
OpenFeature.setProvider(await CloudbeesProvider.build(appKey));
```
</details>
````

<details>
<summary>CloudBees</summary>

```js
import { CloudbeesProvider } from 'cloudbees-openfeature-provider-node';

const appKey = '[YOUR_APP_KEY]';
OpenFeature.setProvider(await CloudbeesProvider.build(appKey));
```
</details>